src/objects/tests/admin/test_core_views.py

```diff
@@ -0,0 +1,69 @@
+from django.urls import reverse
+
+import requests_mock
+from django_webtest import WebTest
+from maykin_2fa.test import disable_admin_mfa
+
+from objects.accounts.tests.factories import UserFactory
+from objects.token.tests.factories import ObjectTypeFactory
+
+from ..utils import mock_objecttype, mock_objecttype_version, mock_service_oas_get
+
+
+@disable_admin_mfa()
+@requests_mock.Mocker()
+class ObjectTypeAdminVersionsTests(WebTest):
+
+    def test_valid_response_view(self, m):
+        objecttypes_api = "https://example.com/objecttypes/v1/"
+        object_type = ObjectTypeFactory.create(service__api_root=objecttypes_api)
+        mock_service_oas_get(m, objecttypes_api, "objecttypes")
+        m.get(f"{objecttypes_api}objecttypes", json=[])
+        m.get(object_type.url, json=mock_objecttype(object_type.url))
+        version = mock_objecttype_version(object_type.url, attrs={"jsonSchema": {}})
+        m.get(
+            f"{object_type.url}/versions",
+            json={
+                "count": 1,
+                "next": None,
+                "previous": None,
+                "results": [version],
+            },
+        )
+
+        user = UserFactory.create(is_staff=True, is_superuser=True)
+
+        # object_type exist
+        url = reverse("admin:objecttype_versions", args=[object_type.pk])
+        response = self.app.get(url, user=user)
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(len(response.json), 1)
+
+        # object_type does not exist
+        url = reverse("admin:objecttype_versions", args=[100])
```

Comment:
Dependent on the amount of objecttypes created in other tests (or using `--keepdb` locally) this PK might be valid, so let's make sure it is unused
```suggestion
        url = reverse("admin:objecttype_versions", args=[object_type.pk+1])
```